tests/dialects/test_duckdb.py

```diff
@@ -583,6 +583,48 @@ def test_bool_or(self):
             write={"duckdb": "SELECT a, BOOL_OR(b) FROM table GROUP BY a"},
         )
 
+    def test_encode_decode(self):
```

Comment:
Let's include `"duckdb": ...` in these tests instead of just spark and presto.

Reply:
Added.